tests/__init__.py

```diff
@@ -2,12 +2,15 @@
 import logging
 import datetime
 import json
+import ast
 from unittest import TestCase
 from contextlib import contextmanager
 
-os.environ['REDASH_REDIS_URL'] = os.environ.get('REDASH_REDIS_URL', "redis://localhost:6379/0").replace("/0", "/5")
-# Use different url for Celery to avoid DB being cleaned up:
-os.environ['REDASH_CELERY_BROKER'] = os.environ.get('REDASH_REDIS_URL', "redis://localhost:6379/0").replace("/5", "/6")
+PYTEST_ENABLE_REDIS_CLEANUP = ast.literal_eval(os.environ.get('PYTEST_ENABLE_REDIS_CLEANUP', 'True'))
```

Comment:
Please use `true` and `redash.settings.parse_boolean` to be consistent with other env variables we have.

But I don't see the value in adding this setting. It is useful locally, because when you use the same Redis server both for tests and development it's annoying (and causes issues) when Redis gets flushed.

The database replacement method here can be more elaborate, but because it's always possible to override with env variables, I figured we could just aim at the default.